settings/preview-deployments.mdx

```diff
@@ -5,20 +5,51 @@ icon: "scan-eye"
 ---
 
 <Info>
-  Preview Deployments are available on the [Growth and Enterprise plans](https://mintlify.com/pricing?ref=preview-deployments). Please{" "}
+  Preview deployments are available on [Growth and Enterprise plans](https://mintlify.com/pricing?ref=preview-deployments). Please{" "}
   <a href="mailto:sales@mintlify.com">contact sales</a> for more information.
 </Info>
 
-Preview Deployments allow you to preview changes to your docs in a live deployment without merging those changes to your production branch.
+Preview deployments let you see how changes to your docs will look before merging to production. Each preview creates a shareable URL that updates automatically as you push new changes.
 
-If you have created a pull or merge request when using Git, the generated preview URL will be available as a comment from the Mintlify bot in the PR. This URL will always reflect the latest deployment changes.
+## Creating preview deployments
 
-## Sharing Preview Deployments
+Preview deployments are created automatically through pull requests or manually from your dashboard.
 
-After generating a preview URL, you can share that link with anybody for them to view.
+### Automatic previews
 
-### Authenticated Preview Deployments
+When you create a pull request, the Mintlify bot automatically adds a link to view the preview deployment in your pull request. The preview updates each time you push new commits to the branch.
 
-If you'd like to restrict access to your Preview Deployments, you can set up preview deployment authentication. When using this authentication method, only Mintlify authenticated users will be able to access your Preview Deployments.
+<Frame>
+  <img src="/images/previews/preview-deployment-light.png" alt="Link to view deployment in the pull request timeline" className="block dark:hidden" />
+  <img src="/images/previews/preview-deployment-dark.png" alt="Link to view deployment in the pull request timeline" className="hidden dark:block" />
+</Frame>
 
-To set up preview deployment authentication, please [contact sales](mailto:sales@mintlify.com).
+### Manual previews
+
+You can manually create a preview for any branch.
+
+1. Go to your [dashboard](https://dashboard.mintlify.com/).
+2. Select **Previews**.
```

Comment:
Maybe add a note that you wont see "Previews" if you arent paying for the Preview deployment add-on or if you arent on the growth plan+